resources/lang/ja.json

```diff
@@ -0,0 +1,401 @@
+{
+  "Actions": "アクション",
+  "Details": "詳細",
+  "If you did not request a password reset, no further action is required.": "もしパスワードのリセットをリクエストしていないなら, これ以上のアクションは必要ありません.",
+  "Reset Password": "パスワードをリセット",
+  "Sorry! You are not authorized to perform this action.": "申し訳ありません! このアクションを実行する権限がありません.",
+  "You are receiving this email because we received a password reset request for your account.": "あなたのアカウントでのパスワードリセットのリクエストがありましたのでこのメールを送っています.",
+  "Confirm Password": "パスワード確認",
+  "Dashboard": "ダッシュボード",
+  "Email Address": "eメールアドレス",
+  "Forgot Your Password?": "パスワードをお忘れですか?",
+  "Forgot your password?": "パスワードをお忘れですか?",
+  "Login": "ログイン",
+  "Logout": "ログアウト",
+  "Password": "パスワード",
+  "Remember Me": "私を覚えておく",
+  "Resources": "リソース",
+  "Send Password Reset Link": "パスワードリセットのリンクを送る",
+  "Welcome Back!": "おかえりなさい!",
+  "Delete Resource": "リソースの削除",
+  "Delete": "削除",
+  "Detach Resource": "リソースの切り離し",
+  "Detach": "切り離し",
+  "Detach Selected": "選択済みの切り離し",
+  "Delete Selected": "選択済みの削除",
+  "Force Delete Selected": "選択済みの強制削除",
+  "Restore Selected": "選択済みの復元",
+  "Restore Resource": "リソースの復元",
+  "Restore": "復元",
+  "Force Delete Resource": "リソースの強制削除",
+  "Force Delete": "強制削除",
+  "The :resource was created!": ":resource が作成されました!",
+  "Are you sure you want to delete this resource?": "本当にこのリソースを削除しますか?",
+  "Are you sure you want to delete the selected resources?": "本当にこの選択済みのリソースを削除しますか?",
+  "Are you sure you want to detach this resource?": "本当にこのリソースを切り離しますか?",
+  "Are you sure you want to detach the selected resources?": "本当にこの選択済みのリソースを切り離しますか?",
+  "Are you sure you want to force delete this resource?": "本当にこのリソースを強制削除しますか?",
+  "Are you sure you want to force delete the selected resources?": "本当にこの選択済みのリソースを強制削除しますか?",
+  "Are you sure you want to restore this resource?": "本当にこのリソースを復元しますか?",
+  "Are you sure you want to restore the selected resources?": "本当にこの選択済みのリソースを復元しますか?",
+  "No :resource matched the given criteria.": "いいえ :resource は与えられた基準にマッチします.",
+  "Another user has updated this resource since this page was loaded. Please refresh the page and try again.": "他のユーザーがこのリソースを更新したのでページがロードされます. ページをリフレッシュして再度お試しください.",
+  "Are you sure you want to delete this file?": "本当にこのファイルを削除しますか?",
+  "Are you sure you want to run this action?": "本当にこのアクションを実行しますか?",
+  "Attach": "アタッチ",
+  "Attach & Attach Another": "アタッチ & もう一つアタッチ",
+  "Cancel": "キャンセル",
+  "Choose": "選択",
+  "Choose File": "ファイルを選択",
+  "Choose Type": "タイプを選択",
+  "Choose an option": "オプションを選択",
+  "Click to choose": "クリックして選択",
+  "Reset Filters": "フィルタをリセット",
+  "Create": "作成",
+  "Create & Add Another": "作成 & もう一つ追加",
+  "Delete File": "ファイル削除",
+  "Edit": "編集",
+  "Edit Attached": "アタッチされたものを編集",
+  "Go Home": "ホームへ",
+  "Hold Up!": "手を上げろ!",
```

Comment:
@Tsumagari Are you sure about the tone of this translation? Is it friendly enough?

If it translates as "Put your hands up!", I was worried that it sounds more aggressive, like if the police were pointing a gun at a bank robber or someone was trying to rob you: ![手を上げろ!](http://1.bp.blogspot.com/_D9rJtdybYUQ/TSKbzkeirrI/AAAAAAAAAGs/y10-3QDlKkc/s1600/put+your+hands+up+copy.jpg)  😆

Reply:
I'm sorry for delaying reply!
I suspect I have misunderstood the mean of "Hold up". Does it similar to "Hold on"?
If it does, I suggest "ちょっと待った!" which means "wait a sec!". This phrase is casual and often used  in conversations theatrically. It is used very commonly in drama or anime, when they want to interrupt or call to stop someone.
```suggestion
  "Hold Up!": "ちょっと待った!",
```

Reply:
Hi @Tsumagari. Thanks, I think that tone of voice sounds just right!

We can't change the file in this PR because I have already merged it, but I will change this line in the master branch. Thank you for taking the time to answer my questions and for contributing Japanese to our package!